### PR TITLE
Add spanish translation for-2019 09 14 rodolfo novak coldcard mk3 and file _index.es.md

### PR DESCRIPTION
**Pull request by BlueeeMoon**

### CODE REVIEW

File: baltic-honeybadger/2019/2019-09-14-rodolfo-novak-coldcard-mk3.es.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+---
+title: Rodolfo Novak Coldcard Mk3 (2019-09-14)
+transcript_by: Michael Folkson
+translation_by: Blue Moon
+categories: ['conferencia']
+tags: ['seguridad', 'cartera hardware', 'cartera']
+---
+
+Name: Rodolfo Novak
```
Comment (adamjonas) [inline baltic-honeybadger/2019/2019-09-14-rodolfo-novak-coldcard-mk3.es.md:9]: this should be in spanish yes?

Comment (BlueeeMoon) [reply baltic-honeybadger/2019/2019-09-14-rodolfo-novak-coldcard-mk3.es.md:9]: > this should be in spanish yes?

It´s correct.

File: baltic-honeybadger/2019/2019-09-14-rodolfo-novak-coldcard-mk3.es.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+Topic: Coldcard Mk3 - Security in Depth
+
+Location: Baltic Honeybadger 2019
+
+Date: 14 de septiembre de 2019 
+
+Audio: https://youtu.be/2IpZWSWUIVE?t=23739
```
[the remark this answers is not in the document]
Comment (BlueeeMoon) [reply baltic-honeybadger/2019/2019-09-14-rodolfo-novak-coldcard-mk3.es.md:17]: > This is video yes?

Yes.

[the remark this answers is not in the document]
Comment (adamjonas) [reply baltic-honeybadger/2019/2019-09-14-rodolfo-novak-coldcard-mk3.es.md:17]: It probably makes more sense to update to`vídeo` then. Would you mind making that update and translating the others as well to be consistent with the other Spanish transcripts?

**Pull request merged**